.github/blunderbuss.yml

```diff
@@ -39,3 +39,13 @@ assign_issues_by:
   - 'api: spanner'
   to:
   - skuruppu
+- labels:
+  - 'api: cloudiot'
+  to:
+  - attilagargyan
+
+assign_prs_by:
```

Comment:
If you add to `CODEOWNERS`, you can skip this. But, it doesn't hurt to have it here, too.

Reply:
thanks!